docs/index.rst

```diff
@@ -26,9 +26,9 @@ Documentation
     :maxdepth: 2
 
     general/index
+    usage/index
```

Comment:
can you move the `usage` section to general? so we have "General > Usage" after installation or so